sips/sip-139.md

```diff
@@ -0,0 +1,81 @@
+
+There is currently an issue where when a synth has been suspended due to [SIP-65](./sip-65.md) and if the new rate is legitimate (and not some fault of a faulty oracle or pricing mechanism) and remains at this new rate (rather than spiking and returning), then resuming a synth will likely result in it being suspended again next exchange - unless the price shifts significantly back towards when the last trade happened.
+
+## Specification
+
+<!--The specification should describe the syntax and semantics of any new feature, there are five sections
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+
+<!--This is a high level overview of *how* the SIP will solve the problem. The overview should clearly describe how the new feature will be implemented.-->
+
+Add an additional function to `Exchanger` that resets the `lastExchangeRate[currencyKey]` to the latest rate from the `ExchangeRates` contract.
+
+### Rationale
+
+<!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+
+While it could be argued that the `SystemStatus.resumeSynth()` function be in charge of resetting this rate, the `SystemStatus` contract has been designed to not know the rest of the system, and is purely holding state. In addition, there's a function `Exchanger.setLastExchangeRateForSynth()` that only `ExchangeRates` can invoke, which could be exposed to the owner in `ExchangeRates`, however updating `ExchangeRates` involves a more complex deployment and is not preferable at this time.
+
+### Technical Specification
+
+<!--The technical specification should outline the public API of the changes proposed. That is, changes to any of the interfaces Synthetix currently exposes or the creations of new ones.-->
+
+```solidity
+
+interface IExchanger {
+
+    function resetLastExchangeRate(bytes32 currencyKey) onlyOwner;
+
+}
+```
+
+### Test Cases
+
+<!--Test cases for an implementation are mandatory for SIPs but can be included with the implementation..-->
+
+- When `resetLastExchangeRate` is invoked by a non-owner, then it always reverts.
+- When `resetLastExchangeRates` is invoked by an owner for `bytes32 currencyKey`, then it updates `lastExchangeRate[currencyKey]` to `ExchangeRates.rateForCurrency(currencyKey)`.
```

Comment:
@justinjmoses  Would there be a use case for an array of keys to reset in one tx instead of multiple?